widget/select.go

```diff
@@ -107,20 +110,67 @@ type Select struct {
 	PlaceHolder string
 	OnChanged   func(string) `json:"-"`
 
-	hovered, tapped bool
-	popUp           *PopUpMenu
+	hovered bool
+	focused bool
+	tapped  bool
+	popUp   *PopUpMenu
+	Parent  fyne.CanvasObject
```

Comment:
This does not seem to be used anywhere and should not be needed

Reply:
I agree.